shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -37,9 +37,8 @@ - (void)testLookUpCallInitiated {
   XCTestExpectation* presentExpectation =
       [self expectationWithDescription:@"Look Up view controller presented"];
 
-  FlutterViewController* engineViewController = [[FlutterViewController alloc] initWithEngine:engine
-                                                                                      nibName:nil
-                                                                                       bundle:nil];
+  FlutterViewController* engineViewController =
+      [[[FlutterViewController alloc] initWithEngine:engine nibName:nil bundle:nil] autorelease];
```

Comment:
The view controller is leaked, causing the newly added notification tests in other testing files to fail.